crates/router/src/core/payments/operations/payment_confirm.rs

```diff
@@ -576,6 +576,10 @@ impl<F: Send + Clone, Ctx: PaymentMethodRetrieve>
             None => None,
         };
 
+        payment_attempt.payment_method_billing_address_id = payment_method_billing
+            .as_ref()
+            .map(|payment_method_billing| payment_method_billing.address_id.clone());
+
```

Comment:
Duplicate assignment operation.

Reply:
Added here while resolving a merge conflict, will remove this.